### PR TITLE
[Contains migration] added last_employer_en and last_job_title_en fi…

### PR DESCRIPTION
**Pull request by EmmaSamoilova**
added last_employer_en and last_job_title_en fields to Pep model

### CODE REVIEW

File: business_register/management/commands/add_transaltion_to_last_job.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        for pep in all_pep:
+            if pep.last_employer:
+                pep.last_employer_en = transliterate(
+                    translate_company_type_in_string(pep.last_employer)
+                )
+            if pep.last_job_title:
+                pep.last_job_title_en = transliterate(
+                    translate_last_position_in_string(pep.last_job_title)
+                )
+            pep.save()
```
Comment (roman-tiukh) [inline business_register/management/commands/add_transaltion_to_last_job.py:26]:
```suggestion
        for pep in Pep.objects.filter(last_employer__isnull=False):
            pep.last_employer_en = transliterate(
                translate_company_type_in_string(pep.last_employer)
            )
            pep.save(update_fields=['last_employer_en'])
        for pep in Pep.objects.filter(last_job_title__isnull=False):
            pep.last_job_title_en = transliterate(
                 translate_last_position_in_string(pep.last_job_title)
            )
            pep.save(update_fields=['last_job_title_en'])
```

File: business_register/filters.py [hunk 2]
```diff
@@ -217,7 +217,9 @@ class PepFilterSet(filters.FilterSet):
             ('is_pep', 'is_pep'),
             ('pep_type', 'pep_type'),
             ('last_job_title', 'last_job_title'),
+            ('last_job_title_en', 'last_job_title_en'),
             ('last_employer', 'last_employer'),
+            ('last_job_title_en', 'last_job_title_en'),
```
Comment (roman-tiukh) [inline business_register/filters.py:222]: How about fullame_en ordering?

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/management/commands/add_transaltion_to_last_job.py (hunk 1 above)
Comment (EmmaSamoilova) [reply business_register/management/commands/add_transaltion_to_last_job.py]: done

File: business_register/filters.py (hunk 2 above)
Comment (EmmaSamoilova) [reply business_register/filters.py:223]: done

File: business_register/models/pep_models.py [hunk 3]
```diff
@@ -68,10 +67,18 @@ class Pep(DataOceanModel):
         _('last position'), max_length=340, null=True, db_index=True,
         help_text='Title of the last job in Ukrainian.'
     )
+    last_job_title_en = models.CharField(
+        _('last position'), max_length=340,  blank=True, default='', db_index=True,
+        help_text='Title of the last job in Ukrainian.'
```
Comment (Andrii17s) [inline business_register/models/pep_models.py:72]: Is it in Ukrainian?

File: business_register/models/pep_models.py [hunk 4]
```diff
@@ -68,10 +67,18 @@ class Pep(DataOceanModel):
...
+    )
     last_employer = models.CharField(
         _('last office'), max_length=512, null=True, db_index=True,
         help_text='Last employer in Ukrainian.'
     )
+    last_employer_en = models.CharField(
+        _('last office'), max_length=512, blank=True, default='', db_index=True,
+        help_text='Last employer in Ukrainian.'
```
Comment (Andrii17s) [inline business_register/models/pep_models.py:80]: Is it in Ukrainian?

**Review by Andrii17s — CHANGES_REQUESTED** [review]

File: business_register/models/pep_models.py (hunk 4 above)
Comment (EmmaSamoilova) [reply business_register/models/pep_models.py]: No, thank you. I have done

**Review by Andrii17s — APPROVED** [review]

File: business_register/filters.py (hunk 2 above)
Comment (Iva-khar) [reply business_register/filters.py:223]: 2 times `last_job_title_en`, but no one` last_employer_en`

File: business_register/management/commands/add_transaltion_to_last_job.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
+from django.core.management import BaseCommand
+from business_register.models.pep_models import Pep
+
+from data_ocean.transliteration.utils import transliterate, translate_company_type_in_string, \
+    translate_last_position_in_string
+
+
+class Command(BaseCommand):
+    help = 'Saving a list of cars that are subject to transport tax'
```
Comment (Iva-khar) [inline business_register/management/commands/add_transaltion_to_last_job.py:9]: change help text or remove

**Review by roman-tiukh — APPROVED** [review]

**Review by IegorSoboliev — APPROVED** [review]

**Pull request merged**